cblox/include/cblox/core/tsdf_submap.h

```diff
@@ -5,6 +5,7 @@
 #include <mutex>
 
 #include <Eigen/Geometry>
+#include <ros/ros.h>
```

Comment:
At least in master we wanna keep cblox library ROS independent (as opposed to cblox_ros which adds ros interfaces). I guess this timing code isn't intended for the master however, so it's fine, we just wont merge this.